CardGame.playground/Contents.swift

```diff
@@ -28,6 +81,19 @@ import Foundation
 //: Make the `Card` type conform to the `Comparable` protocol. Implement the `<` and `==` methods such that they compare the ranks of the `lhs` and `rhs` arguments passed in. For the `==` method, compare **both** the rank and the suit.
 
 
+struct Card: CustomStringConvertible, Comparable {
+    static func < (lhs: Card, rhs: Card) -> Bool {
+        return lhs.suit == rhs.suit && lhs.rank == rhs.rank
```

Comment:
This line should compare the values, not assure they are the same. Try this on line 86 instead:

```
return lhs.rank < rhs.rank
```